test/e2e/apps/statefulset.go

```diff
@@ -248,14 +248,8 @@ var _ = SIGDescribe("StatefulSet", func() {
 			sst.WaitForRunningAndReady(*ss.Spec.Replicas, ss)
 		})
 
-		/*
-			Release : v1.9
-			Testname: StatefulSet, Rolling Update
-			Description: StatefulSet MUST support the RollingUpdate strategy to automatically replace Pods one at a time when the Pod template changes. The StatefulSet's status MUST indicate the CurrentRevision and UpdateRevision. If the template is changed to match a prior revision, StatefulSet MUST detect this as a rollback instead of creating a new revision. This test does not depend on a preexisting default StorageClass or a dynamic provisioner.
-		*/
-		framework.ConformanceIt("should perform rolling updates and roll backs of template modifications", func() {
-			By("Creating a new StatefulSet")
-			ss := framework.NewStatefulSet("ss2", ns, headlessSvcName, 3, nil, nil, labels)
+		// This function is used by the following two tests.
+		rollbackTest := func(ss *apps.StatefulSet) {
```

Comment:
Instead of being an inline function, can this just be a named function?

Reply:
Can do, should it just go above this current function?

Reply:
Maybe. It's referencing variables instantiated within the enclosing function, so there would need to be several parameters or a new struct, right? Also, where would it go in the file, ideally?

Reply:
In other test files, I've seen the test functions just go at the bottom